content/en/docs/reference/glossary/cgroup.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: cgroup
+title: control group
```

Comment:
hello @ljnaresh . Sorry for the confusion. I was suggesting that the title be consistent with the other titles that are acronyms or have abbreviations, such as:
`title: RBAC (Role-Based Access Control)`

`title: cgroup (control group)`

Does this make sense? Otherwise, the changes seem fine 😃.

Reply:
Yes, this is fine.